OSDOCS 17653 GA-BYOPKI for image verification in OCP (#103803)
openshift/openshift-docs: modules/nodes-sigstore-configure-cluster-policy.adoc

* You have a sigstore-supported public key infrastructure (PKI) key, a Bring Your Own Public Key Infrastructure (BYOPKI) certificate, or provide a link:https://docs.sigstore.dev/cosign/signing/overview/[Cosign public and private key pair] for signing operations.
* You have a signing process in place to sign your images.

> * You have access to a registry that supports Cosign signatures, if you are using Cosign signatures.

Reviewer: Mirroring the sigstore signatures seems to still be valid guidance for air-gapped or disconnected environments. However, mentioning the Technology Preview feature set here might no longer be relevant. Would it make sense to adjust this section to focus on recommending signature mirroring explicitly for disconnected users? @wking What are your thoughts?

* You have a sigstore-supported public key infrastructure (PKI) key, a Bring Your Own Public Key Infrastructure (BYOPKI) certificate, or provide a link:https://docs.sigstore.dev/cosign/signing/overview/[Cosign public and private key pair] for signing operations.
* You have a signing process in place to sign your images.
* You have access to a registry that supports Cosign signatures, if you are using Cosign signatures.

> * If registry mirrors are configured for the {product-title} release image repositories, `quay.io/openshift-release-dev/ocp-release` and `quay.io/openshift-release-dev/ocp-v4.0-art-dev`, optionally, mirror the sigstore signatures for the {product-title} release images into your mirror registry. Otherwise, the default `openshift` cluster image policy, which enforces signature verification for the release repository, blocks the ability of the Cluster Version Operator to move the CVO pod to new nodes, preventing the node update that results from the feature set change.

Reviewer 2: ```suggestion
* If mirror registry is configured for the {product-title} release image repositories, `quay.io/openshift-release-dev/ocp-release` and `quay.io/openshift-release-dev/ocp-v4.0-art-dev`, optionally, mirror the sigstore signatures for the {product-title} release images into your mirror registry. Otherwise, the default `openshift` cluster image policy, which enforces signature verification for the release repository, blocks the ability of the Cluster Version Operator to move the CVO pod to new nodes, preventing the node update that results from the feature set change.
```
